index.js

```diff
@@ -31,6 +31,7 @@ class Ora {
 		this.stream = this.options.stream;
 		this.id = null;
 		this.frameIndex = 0;
+		this.indent = this.options.indent || 0;
```

Comment:
@sindresorhus I faced this issue today, i have nested prints in terminal and spinner is starting on very first column which is breaking the overall layout of printed information. It is cosmetic change, but at least 2 persons ran into this issue 😄